packages/netlify-cms-core/src/backend.ts

```diff
@@ -351,6 +350,39 @@ export class Backend {
     return entries;
   }
 
+  async listAllMultipleEntires(collection: Collection, page: number, locales: string[]) {
+    const multiContent = collection.get('multi_content');
+    const depth = multiContent === DIFF_FOLDER ? 2 : 0;
+    const entries = await this.listAllEntries(collection, depth);
```

Comment:
`listAllEntries` can be very slow since it is traversing all the pagination links.
Could we read just the default locale ones?
Not an issue with GitHub as the tree API returns 100k entries so we can filter client side and read only the default locale.
For GitLab/BitBucket, not sure if we can filter bases on a path.
I'll probably have to improve `listAllEntries` for the sorting feature.

Reply:
The idea here is that since we are now dealing with multiple files, relying on a paginated list may leave out some translated files that are part of a Multi Content post from the results, thoughts?.

Reply:
Yes, I understand. Leave it for now, I'll have to improve it regardless for the sorting feature.